prog/net-plugin/launch.sh

```diff
@@ -46,6 +46,7 @@ exec /home/weave/weaver $EXTRA_ARGS --port=6783 $(router_bridge_opts) \
     --host-root=/host \
     --proc-path=/host/proc \
     --http-addr=$HTTP_ADDR --status-addr=$STATUS_ADDR \
+    --without-proxy \
```

Comment:
We have `--without-dns`, hence I went for `--without-proxy`. 
However, as discussed offline and summarised [here](https://github.com/weaveworks/weave/pull/2936#discussion_r115501309), we should really flip this and have `--proxy`. This raises the question of `--without-dns` vs. `--dns`, but prob. for another PR.